### PR TITLE
DAG'24

### PR DESCRIPTION
**Pull request by Multipacker**

### CODE REVIEW

File: web/src/config.ts [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import chairmanProfile from "./assets/profiles/chairman.jpg"
 import financialManagerProfile from "./assets/profiles/financialManager.jpg"
-import eventsProfile from "./assets/profiles/events.jpg"
-import logisticsProfile from "./assets/profiles/logistics.jpg"
```
Comment (The1Penguin) [inline web/src/config.ts:4]: Would prefer to keep this commented out so its' easier for future DAG to add a future photo

File: web/src/views/Home.vue [hunk 2]
```diff
@@ -139,7 +139,7 @@ export default defineComponent({
           .text-2xl.text-brand.font-bold.font-display {{ name }}
           .text-lg.font-bold {{ title }}
           .text-lg {{ email }}
-    .grid.grid-cols-1.gap-4(class="lg:grid-cols-4 lg:gap-32")
+    .grid.grid-cols-1.gap-4(class="lg:grid-cols-3 lg:gap-32")
```
Comment (The1Penguin) [inline web/src/views/Home.vue:142]: Does this look better on the web? Don't have laptop access at the moment. Has phone layout been checked out?

**Review by The1Penguin — COMMENTED** [review]
Please answer the comments I have left

File: web/src/views/Home.vue (hunk 2 above)
Comment (Multipacker) [reply web/src/views/Home.vue:142]: This does look a lot better as they are one member fewer this year, without it there is just an empty space when it lists the members. I tried it with firefoxes responsive design and it seems to work.

Comment (The1Penguin) [reply web/src/views/Home.vue:142]: Sounds fair

File: web/src/config.ts (hunk 1 above)
Comment (Multipacker) [reply web/src/config.ts:4]: That sounds fair, I'll add it back in. Eventually I want to improve the documentation for the site to make it clearer.

**Review by The1Penguin — APPROVED** [review]
LGTM

**Pull request merged**